### DISCUSSION TITLE
Plugins or Add Ons?

### DISCUSSION THREAD

**Discussion by DavidG-Aus** [supermerill/SuperSlicer#1870] [Q&A]
I have just made the move from Cura and I am very impressed with SuperSlicer.  
There are a couple of features from Cura that I am missing which were added with plugins. The two features I am missing are:
- a plugin that automatically determined the best orientation to print an object and adjusted each object as imported
- a way to view the webcam stream from OctoPrint
Is there a way to achieve either of the above?
From what I can see, SS does not support plugins or addons. Is that correct?

Cheers,
DG

**Comment by supermerill** [comment]
no, no, yes.
or the webcam stream, i don't really understand the value, as it's just a web page, right? Why using the slicer for that?

For the "best orientation", there is an option in arrange to let the slicer choose the z-rotation to try to fit as much as possible in the plater. But i don't think that's what you want.

**Comment by DavidG-Aus** [comment]
The Auto Orientation plug in tries to work out which way is best to print the object, I guess based on least support required or whatever.  It gets it right about 95% of the time. Just means a couple less clicks required per object placed.
Being able to view the live OctoPrint stream from the slicer means I can have the OctoPrint web page looking at something else or not even open (unwise).
